tests/test_run_feature_vector.py

```diff
@@ -7,22 +7,17 @@
 import pytest
 from dictdiffer import diff
 
+path_to_current_file = os.path.realpath(__file__)
+current_directory = os.path.split(path_to_current_file)[0]
+
+TEST_OUTPUT_DIR = os.path.join(current_directory, "data/feature_vector")
+
+nwb2_file1 = os.path.join(current_directory, "data/Vip-IRES-Cre;Ai14(IVSCC)-226110.03.01.nwb")
+nwb2_file2 = os.path.join(current_directory, "data/Vip-IRES-Cre;Ai14(IVSCC)-236654.04.02.nwb")
+print(f"*****nwbfile path in test_run_feature_vector {nwb2_file1}")
+assert 0
```

Comment:
?

Reply:
it was my way of seeing the path that circleci is using for the files. this will be removed.